apis/Google.Cloud.OsLogin.V1Beta/.repo-metadata.json

```diff
@@ -1,4 +1,4 @@
 {
   "distribution_name": "Google.Cloud.OsLogin.V1Beta",
-  "release_level": "beta"
+  "release_level": "ga"
```

Comment:
This is now a little bit confusing, I get it, it's just confusing.

Reply:
Eek, that shouldn't be there. Will need to fix that.

Reply:
Ah, this is tricky, actually. The repo metadata generator can't know what's *been* released when all it knows is what *won't* be released. It's assuming that because there's a 1.99, there must have been a 1.98.
Humbug. Will need to think about this more.

Reply:
Yes, I was just going to add a comment around this, that is that we don't do 1.1.99-beta00 instead we do 1.2.00-beta00, so why can't we do 2.0.0-alpha00?

Reply:
We could do that - we'd just need to be really careful not to then bump it to 3.0.0 when we do the major version bump.

I would also note that with 2.0.0-alpha00, this would probably *still* say "Ah, there's a GA available". I think we probably need an override for that.